packages/mobile/src/web3/contracts.ts

```diff
@@ -15,8 +15,11 @@ import { provider } from 'web3-core'
 
 const tag = 'web3/contracts'
 
-const contractKitForno = newKitFromWeb3(getWeb3(true))
-const contractKitGeth = newKitFromWeb3(getWeb3(false))
+// TODO(yorke): merge instances of contractkit and refactor retrieval of providers
+const IpcProvider = getIpcProvider()
+export const gethWallet = new RpcWallet(IpcProvider)
+const contractKitForno = newKitFromWeb3(getWeb3(true), gethWallet)
```

Comment:
Previously contractKitForno did not depend on the IPC provider- will this be defined if the app is started in forno mode?

Reply:
hmm, good point
this does require geth is running although we can turn off syncing for the lower end devices
is there a process for testing on lower end hardware?